.github/workflows/post-merge-dynamo-to-builb.yml

```diff
@@ -0,0 +1,53 @@
+name: Deploy Main to Build Env L2Dynamo - cic-cri-ddb
```

Comment:
Typo in the name of this file = builb instead of build.

Reply:
This `name` is also what shows on the github actions UI, so let's name them consistently and clearly.  Can you also create a wiki page with details so that new joiners can understand what they are.